npm/private/test/repositories_checked.bzl

```diff
@@ -1,4 +1,4 @@
-"@generated by @aspect_rules_js//npm/private:npm_translate_lock.bzl from //:pnpm-lock.yaml"
```

Comment:
What caused this to change? Don't want "aspect_rules_js" in there?

Reply:
just a tweak so the repository rule type & name is identified instead of the source file

Reply:
this is all behind the scenes anyway will never seen by the average user